src/cirrus/github_tools.py

```diff
@@ -142,14 +142,18 @@ def get_commit_msgs(owner, repo, since_sha, token=None):
     resp.raise_for_status()
     result = []
     for commit in resp.json():
+
+        blame = None
         if commit.get("committer"):
-            committer = commit['committer']['login']
-        else:
+            blame = commit['committer'].get('login')
+
+        if blame is None:
```

Comment:
The reason have it as a None check, is because it also handles the case that `'login` is None.